_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -964,6 +964,12 @@ private function _process_onboarding( $data ) {
 		$site_title = get_option( 'blogname' );
 		$author = get_current_user_id() || 1;
 
+		if ( ! empty( $data['siteType'] ) ) {
+			if ( ! ( update_option( 'jpo_site_type', $data['siteType'] ) || get_option( 'jpo_site_type' ) == $data['siteType'] ) ) {
```

Comment:
Can this option be a Jetpack Option instead or prefixed with `jetpack_`?

Reply:
Unfortunately no, because we're mirroring how the plugin used to work, and mirroring the exact same option that it used to store the site type, see https://github.com/Automattic/jetpack-onboarding/blob/master/class.jetpack-onboarding-end-points.php#L7

Reply:
Got it! Thanks for explaining @ebinnion do you have an opinion about how this impact the cleanup process on deactivation ?

Reply:
If we want to remove `jpo_site_type` on deactivation, then I'd add `jpo_site_type` to the array returned in `Jetpack_Options::get_all_wp_options()`. 

`Jetpack_Options::delete_all_known_options()` is called when Jetpack is uninstalled, and it will all Jetpack options that are stored in `jetpack_options` and `jetpack_private_options` as well as the options returned from `Jetpack_Options::get_all_wp_options()`.

Reply:
Thanks @ebinnion! Added in https://github.com/Automattic/jetpack/pull/8431/commits/74309d2150025b42b8382f25c1c03551c22208ad.